en/docs/setup/migrating-to-590.md

```diff
@@ -115,6 +115,25 @@ sections to read the prerequisites.
     `           <NEW_IS_HOME>/repository/resources/security          `
     directory.
 
+    !!! note
+        In WSO2 Identity Server 5.9.0, it is required to use a certificate with the RSA key size greater than 2048. If you have used a certificate that has a weak RSA key (key size less than 2048) in the previous version, you need to add the following configuration to `<NEW_IS_HOME>/repository/conf/deployment.toml` file to configure internal and primary keystores. You should point the internal keystore to the keystore copied from the previous IS version and the primary keystore can be pointed to a keystore with a certificate, which has strong RSA key. 
+
+        ```toml
+        [keystore.primary]
+        file_name = "primary.jks"
+        type = "JKS"
+        password = "wso2carbon"
+        alias = "wso2carbon"
+        key_password = "wso2carbon"
+
+        [keystore.internal]
+        file_name = "internal.jks"
+        type = "JKS"
+        password = "wso2carbon"
+        alias = "wso2carbon"
+        key_password = "wso2carbon"
+        ```
```

Comment:
How about the following:

... in the previous section, to configure internal and primary keystores, add the following configuration to the `deployment.toml` file in the `<NEW_IS_HOME>/repository/conf` directory.

```
<configs>
```
Make sure to point the internal keystore to the keystore that is copied from the previous WSO2 Identity Server version. The primary keystore can also be pointed to a keystore with a certificate that has a strong RSA key.

Reply:
Fixed with d39d8f088d407fe7aafb2f9c93c3b56dc213a66a